recipes/ipython-memory-usage/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ipython-memory-usage" %}
+{% set version = "1.1" %}
+{% set sha256 = "1f4697210257f853fea74de0cf3fae60a32e550e578bac6f46de9b40c550422b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ianozsvald/ipython_memory_usage.git
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+
+about:
+  home: https://github.com/ianozsvald/ipython_memory_usage
+  license: BSD
+  license_file: LICENSE
+  summary: 'IPython tool to report memory usage deltas for every command you type. If you are running out of RAM then use this tool to understand what’s happening. It also records the time spent running each command.'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    IPython tool to report memory usage deltas for every command you type.
+    If you are running out of RAM then use this tool to understand what’s
+    happening. It also records the time spent running each command.
+  doc_url: http://simplejson.readthedocs.io/
+  dev_url: https://github.com/ianozsvald/ipython_memory_usage
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - pmalde
+    - ianozsvald
```

Comment:
@ianozsvald -- are you ok being added as a co-maintainer of the `ipython_memory_usage` recipe on conda-forge?

http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role